esti/system_test.go

```diff
@@ -344,6 +345,24 @@ func requireBlockstoreType(t testing.TB, requiredTypes ...string) {
 	}
 }
 
-func isBasicAuth() bool {
-	return viper.GetBool("auth.basic")
+func isBasicAuth(t testing.TB, ctx context.Context) bool {
+	t.Helper()
+	return getRBACState(t, ctx) == "none"
+}
+
+func isAdvancedAuth(t testing.TB, ctx context.Context) bool {
```

Comment:
The ACL server configuration returns `true` for this? If so, it's quite confusing as ACL isn't the advanced auth that we got (it's not RBAC)

Reply:
No it does not. It return false